README.md

```diff
@@ -6,27 +6,26 @@
 
 Resolve and convert CSS colors.
 
-
 ## Install
 
 ```console
 npm i @asamuzakjp/css-color
 ```
 
-
 ## Usage
 
 ```javascript
-import { convert, isColor, resolve } from '@asamuzakjp/css-color';
+import { convert, isColor, resolve } from '@asamuzakjp/css-color'
 
-const resolvedValue =
-  resolve('color-mix(in oklab, lch(67.5345 42.5 258.2), color(srgb 0 0.5 0))');
+const resolvedValue = resolve(
+  'color-mix(in oklab, lch(67.5345 42.5 258.2), color(srgb 0 0.5 0))',
```

Comment:
bring back semi.

Reply:
I understand. No need to comment repeatedly for the same style of issue

Reply:
Got it, thanks.